### ISSUE TITLE
Unable to retrieve list of customers

### ISSUE THREAD

**Issue by JamesCoonce** [vapor-community/stripe-provider#64]
Unable to retrieve a list of customers.

Error:
``` {
"error": true,
"reason": "Value required for key 'total_count'."
}```

My code is 
```
    func index(_ req: Request) throws -> Future<StripeCustomersList> {
        let client = try req.make(StripeClient.self)
        print(client)
        return try client.customer.listAll()
    }
```

**Comment by Andrewangeta** [comment]
I’ll get an update out today fixing these issues.

**Comment by JamesCoonce** [comment]
Thanks, I appreciate it. I'd like to start writing documentation soon along with a few tutorials

**Comment by anthonycastelli** [comment]
@Andrewangeta Have you looked into this one?

**Comment by Andrewangeta** [comment]
it's the optional stuff @anthonycastelli  all should be fixed in 2.2.0 release.

**Comment by anthonycastelli** [comment]
This is now fixed and will be addressed in the upcoming release.
